src/main/resources/migration/changelog-v5.3.0.xml

```diff
@@ -2470,4 +2470,24 @@
                 ON "COMPONENT" USING GIN ("DIRECT_DEPENDENCIES" GIN_TRGM_OPS);
         </sql>
     </changeSet>
+
+    <changeSet id="v5.3.0-15" author="sahibamittal">
+        <createTable tableName="PROJECT_METADATA">
+            <column autoIncrement="true" name="ID" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PROJECT_METADATA_PK"/>
+            </column>
+            <column name="PROJECT_ID" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="SUPPLIER" type="TEXT"/>
+            <column name="AUTHORS" type="TEXT"/>
+        </createTable>
+        <addColumn tableName="COMPONENT">
+            <column name="SUPPLIER" type="TEXT"/>
+        </addColumn>
+        <addColumn tableName="PROJECT">
+            <column name="SUPPLIER" type="TEXT"/>
+            <column name="MANUFACTURER" type="TEXT"/>
+        </addColumn>
```

Comment:
I think we should take advantage of the fact we can focus on PostgreSQL now, and use the [`JSONB`](https://www.postgresql.org/docs/current/datatype-json.html) type here, instead of `TEXT`. 

Makes it more efficient to store, query, and potentially index if we end up needing that. We already use `JSONB` for some of the `VulnerabilityPolicy` fields, so it *should just work* :D https://github.com/DependencyTrack/hyades-apiserver/blob/4e413d9bf0a77d594bcd327338ecb6c05e8a292c/src/main/resources/migration/changelog-v5.3.0.xml#L2328-L2350